padl/dumptools/symfinder.py

```diff
@@ -607,39 +608,51 @@ def f(x):
     scope: Scope
     n: int = 0
 
+    def __post_init__(self):
+        self.variants = [(self.name, self.n)]
+
+    def add_variant(self, name, n):
+        self.variants.append((name, n))
+
     def __hash__(self):
         return hash((self.name, self.scope, self.n))
 
     def __eq__(self, other):
         return (self.name, self.scope, self.n) == (other.name, other.scope, other.n)
 
+    def copy(self):
+        _copy = type(self)(self.name, self.scope, self.n)
+        _copy.variants = [(_name, _n) for _name, _n in self.variants]
+        return _copy
+
 
 def find_in_scope(scoped_name: ScopedName):
     """Find the piece of code that assigned a value to the variable with name
     *scoped_name.name* in the scope *scoped_name.scope*.
 
     :param scoped_name: Name (with scope) of the variable to look for.
+    :return: Tuple as ((source, node), scope, name), where
+        * source: String representation of piece of code.
+        * node: Ast node for the code.
+        * scope: Scope of the code.
+        * name: Name of variable (str).
+
     """
     scope = scoped_name.scope
-    i = scoped_name.n
+    n = scoped_name.n
```

Comment:
I think you can remove this line. It looks like `n` is not used in this function.

Reply:
removed!